schemas/Manual.schema.catalog.json

```diff
@@ -1,42 +1,42 @@
 {
-  "$schema": "https://json.schemastore.org/schema-catalog.json",
-  "schemas": [
-    {
-        "name": "Manual items.json",
-        "description": "items.json schema file for the manual Archipelago Apworld",
-        "fileMatch": [ "**/manual*/data/items.json" ],
-        "url": "https://raw.githubusercontent.com/ManualForArchipelago/Manual/main/schemas/Manual.items.schema.json"
-    },
-    {
-        "name": "Manual game.json",
-        "description": "game.json schema file for the manual Archipelago Apworld",
-        "fileMatch": [ "**/manual*/data/game.json" ],
-        "url": "https://raw.githubusercontent.com/ManualForArchipelago/Manual/main/schemas/Manual.game.schema.json"
-    },
-    {
-        "name": "Manual locations.json",
-        "description": "locations.json schema file for the manual Archipelago Apworld",
-        "fileMatch": [ "**/manual*/data/locations.json" ],
-        "url": "https://raw.githubusercontent.com/ManualForArchipelago/Manual/main/schemas/Manual.locations.schema.json"
-    },
-    {
-        "name": "Manual regions.json",
-        "description": "regions.json schema file for the manual Archipelago Apworld",
-        "fileMatch": [ "**/manual*/data/regions.json" ],
-        "url": "https://raw.githubusercontent.com/ManualForArchipelago/Manual/main/schemas/Manual.regions.schema.json"
-    },
-    {
-        "name": "Manual categories.json",
-        "description": "categories.json schema file for the manual Archipelago Apworld",
-        "fileMatch": [ "**/manual*/data/categories.json" ],
-        "url": "https://raw.githubusercontent.com/ManualForArchipelago/Manual/main/schemas/Manual.categories.schema.json"
-    },
-    {
-        "name": "Manual meta.json",
-        "description": "meta.json schema file for the manual Archipelago Apworld",
-        "fileMatch": [ "**/manual*/data/meta.json" ],
-        "url": "https://raw.githubusercontent.com/ManualForArchipelago/Manual/main/schemas/Manual.meta.schema.json"
-    }
-  ],
-  "version": 20240406.02
+    "$schema": "https://json.schemastore.org/schema-catalog.json",
+    "schemas": [
+        {
+            "name": "Manual items.json",
+            "description": "items.json schema file for the manual Archipelago Apworld",
```

Comment:
I think I missed this when it came in initially, but "schema file for <referring to Manual> Apworld" should be "schema file for <referring to Manual> APWorlds" or "schema file for APWorlds created with <referring to Manual>". (For referring to Manual, see the above comment about the two possible variations to use. 😄)

This is because "the APWorld" is too small in scope when referring to Manual in the context of it being an APWorld templating system, which can occasionally be used to create APWorld templates (like Axx's fighting game one).